src/new-components/columns/columns.tsx

```diff
@@ -23,11 +23,12 @@ type ColumnWidth =
     | '4/5'
 
 interface ColumnProps {
+    height?: 'full' | 'content'
     width?: ColumnWidth
 }
 
 const Column = polymorphicComponent<'div', ColumnProps>(function Column(
-    { width = 'auto', children, exceptionallySetClassName, ...props },
+    { width = 'auto', height = 'content', children, exceptionallySetClassName, ...props },
```

Comment:
@gnapse I wonder if we should default to `full` to retain the previous behaviour? I'm not sure whether we have use cases that relied on the columns to be equal in height, but it may not be easy to discover them all across Todoist and Twist.

Reply:
@frankieyan ignore this PR for now and do not release it. I need to think more about it. I'm inclined to think that we may not need this, and this should be solved seamlessly by the system.

There's no rush to make a decision on this, as it is solving an issue that I myself raised, and there's no user reports on it.

Reply:
@frankieyan can you please add a review requesting changes? I cannot do it as the PR author.